integration/test/API/BrigInternal.hs

```diff
@@ -43,3 +44,70 @@ createUser domain cu = do
                  | cu.team
                ]
         )
+
+data FedConn = FedConn
+  { domain :: String,
+    searchStrategy :: String
+  }
+  deriving (Eq, Ord)
+
+instance ToJSON FedConn where
+  toJSON (FedConn d s) =
+    Aeson.object
+      [ "domain" .= d,
+        "search_policy" .= s
+      ]
+
+instance FromJSON FedConn where
+  parseJSON = withObject "FedConn" $ \obj -> do
+    FedConn
+      <$> obj .: fromString "domain"
+      <*> obj .: fromString "search_policy"
+
+createFedConn :: HasCallStack => FedConn -> App Response
+createFedConn fedConn = do
+  res <- createFedConn' fedConn
+  res.status `shouldMatchRange` (200, 299)
+  pure res
+
+createFedConn' :: HasCallStack => FedConn -> App Response
+createFedConn' fedConn = do
+  owndom <- ownDomain
+  req <- rawBaseRequest owndom Brig Unversioned "/i/federation/remotes"
```

Comment:
No need to bind the domain, you can just pass it directly

```suggestion
  req <- rawBaseRequest ownDomain Brig Unversioned "/i/federation/remotes"
```
Also the domain should not be hardcoded in the API.* Modules like this one, so that the api function can be used in other tests as well, see uses of the `objDomain` function